### DISCUSSION TITLE
A "Made with Lowdefy" badge

### DISCUSSION THREAD

**Discussion by astrogewgaw** [lowdefy/lowdefy#708] [Ideas and feature requests]
It would be nice to have a "Made with Lowdefy" badge, with the Lowdefy logo, that can be put into a project README. Ideally, it should support all the styles from **https://shields.io**: _plastic_, _flat_, _flat-square_, _for-the-badge_, and _social_. It would definitely help advertise the project more 😁 👍🏾 !

**Comment by Gervwyk** [comment]
Hi @astrogewgaw 
This is a cool idea! I'll investigate and make it happen in the near future ⚡️

**Reply by astrogewgaw** [reply]
Awesome @Gervwyk 👍🏾 ! If I come up with something, I will let you know 😁 .
